.github/workflows/check.yml

```diff
@@ -82,7 +82,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: hack
-          args: --feature-powerset check --all-targets
+          args: check --feature-powerset --all-targets
```

Comment:
Why this change? I though hack arguments had to come before the cargo subcommand?

Reply:
All the docs for the cargo hack command show them being called in this order.. "check --feature-powerset etc...).